doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.md

```diff
@@ -310,6 +310,20 @@ Refer to [this section](kuberay-external-storage-namespace-example) in the earli
 
 * `ENABLE_GCS_FT_REDIS_CLEANUP`: The feature gate `ENABLE_GCS_FT_REDIS_CLEANUP` is true by default, and users can turn if off by setting the environment variable in [KubeRay operator's Helm chart](https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/values.yaml).
 
+```{admonition} Setup Key Eviction on Redis
+If users turn `ENABLE_GCS_FT_REDIS_CLEANUP` off but still want GCS metadata to be removed automatically,
```

Comment:
```suggestion
If you disable `ENABLE_GCS_FT_REDIS_CLEANUP` but still want Redis to remove GCS metadata automatically,
```

Reply:
Is this what you mean?

Reply:
Yes, it is. Thank you for clarification and I think removing the word “still” will be better.